codec-haproxy/src/main/java/io/netty/handler/codec/haproxy/HAProxyMessage.java

```diff
@@ -182,26 +202,28 @@ static HAProxyMessage decodeHeader(ByteBuf header) {
                 addressLen = addressEnd - startIdx;
             }
             dstAddress = header.toString(startIdx, addressLen, CharsetUtil.US_ASCII);
+            //AF_UNIX defines that excactly 108 bytes are reserved for the address
```

Comment:
unfortunately the reader index is not updated before (the toString method doesn't modify the reader index and the forEachByte method also doesn't modify the reader index. If we don't update the index, we're starting to read on the wrong place for the TLVs